### PR TITLE
core API

### PR DESCRIPTION
**Pull request by awiouy**

### CODE REVIEW

**Comment by sashahilton00** [comment]
Why has keymaster been removed?

**Comment by awiouy** [comment]
Nothing refers to it

**Comment by awiouy** [comment]
Thanks for the explanation. I dropped the commit.

**Comment by awiouy** [comment]
I am done for today.
Please review, and merge if ok.

**Comment by sashahilton00** [comment]
Will do, thanks

File: protocol/src/mercury.rs [hunk 1]
```diff
@@ -50,17 +50,17 @@ impl MercuryMultiGetRequest {
 
     // repeated .MercuryRequest request = 1;
```
Comment (plietar) [inline protocol/src/mercury.rs]: This file is generated automatically from the protobuf files. It shouldn't be modified.

Comment (awiouy) [reply protocol/src/mercury.rs]: Thanks for the explanation. I dropped the commit.

**Pull request merged**